plugins/modules/zos_data_set.py

```diff
@@ -24,7 +24,9 @@
 description:
   - Create, delete and set attributes of data sets.
   - When forcing data set replacement, contents will not be preserved.
-author: "Blake Becker (@blakeinate)"
+author:
+  - "Blake Becker (@blakeinate)"
+  - "Rich Parker (@richp405)"
```

Comment:
Yeah you deserve to be up there after dealing with so many showstoppers.

Reply:
Thank you!